### DISCUSSION TITLE
Support of python 3.10

### DISCUSSION THREAD

**Discussion by VDuchauffour** [facebookresearch/pytorch3d#1389] [Q&A]
Any idea about support of python 3.10 ? Same question about Torch 1.13.
Thanks

**Comment by bottler** [comment]
We have builds for Python 3.10 already. The next release will have builds for PyTorch 1.13.0, which has come out since the current release.

**Reply by VDuchauffour** [reply]
Ok thanks for the quick answer! Any idea when the new version will be released?
